index.d.ts

```diff
@@ -2349,9 +2352,6 @@ declare module 'mongoose' {
 
     /** Appends new custom $unwind operator(s) to this aggregate pipeline. */
     unwind(...args: any[]): this;
-
-    /** Appends new custom $project operator to this aggregate pipeline. */
-    project(arg: any): this
```

Comment:
This shouldn't be a blocker, but I wonder how this change got here... Merge issue of some sort?